deps/llvm.mk

```diff
@@ -441,6 +441,7 @@ $(eval $(call LLVM_PATCH,llvm-D27609-AArch64-UABS_G3)) # Remove for 4.0
 $(eval $(call LLVM_PATCH,llvm-D27629-AArch64-large_model))
 # patches for NVPTX
 $(eval $(call LLVM_PATCH,llvm-NVPTX-addrspaces))
+$(eval $(call LLVM_PATCH,LLVM-D37576-NVPTX-sm_70)) # Remove for 6.0
```

Comment:
Would be better to move this to the bottom (patch apply order can get upset for an existing build) and include it on 0.4 as well. I'm still testing 5.0 patch set so that can wait.